pyproject.toml

```diff
@@ -114,4 +114,4 @@ exclude = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "-n auto --timeout 700"
+addopts = "-n auto --timeout 700 --durations=10"
```

Comment:
This will print the slowest 10 tests, which means that if someone gets annoyed by how slow the tests are, they know where to start optimizing. :D